### ISSUE TITLE
how to generate segmentation masks?

### ISSUE THREAD

**Issue by Usman-Rafique** [angelwmab/Time-Flies-Animating-a-Still-Image-with-Time-Lapse-Video-as-Reference#1]
Nice work and thanks for releasing the code.

Can you please share how to generate segmentation labels for training and evaluation? The ADE20K dataset (cited in the paper) has 150 classes but `/util/utils.py` suggests that there are 8 classes. 

Can you share the code of converting ADE20K classes to the desired 8-channel mask for the method?

**Comment by angelwmab** [comment]
Thank you for your question.
For the segmentation maps of the training set, we annotated them by ourselves, and you are able to download them [here](https://drive.google.com/drive/folders/1_RGhDdLSpdrb_bk0x-EkXz9Jmhm3AQHY).
As for the testing segmentation maps, they are generated by [this github](https://github.com/CSAILVision/semantic-segmentation-pytorch) and then manually remove some wrong tags. So sorry we don't have a code to convert them.
But I think our model can also support more classes if you add new mask in `MaskHelper(seg,color)` in `/util/utils.py`

**Comment by Usman-Rafique** [comment]
I see, no worries. Thanks for the explanation!
